src/pages/ProductDetail/DetailTop/Options/Options.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { ImLocation } from 'react-icons/im';
+
+function Options({ item }) {
+  const [option, setOption] = useState({});
+
```

Comment:
```suggestion
const itemOption = {
    상품상태: 'condition',
    교환여부: 'exchangable',
    거래지역: 'region',
  };

  const itemText = {
    condition: '중고',
    exchangable: '교환불가능',
    region: '전국',
  };
```

3항 연산자를 중첩해서 사용하는 방법보다
객체를 만들어서 mapping을 하는 방법이 추후 유지보수나
클린한 코드에도 도움이 되겠네요!